docs/ref/migration-operations.txt

```diff
@@ -195,15 +195,12 @@ if the field is nullable or if it has a default value that can be used to
 populate the recreated column. If the field is not nullable and does not have a
 default value, the operation is irreversible.
 
-.. admonition:: PostgreSQL
```

Comment:
> I've added test but not sure about "cleanliness" as I've discovered that mysql is not restricting column drop used by view 😬. 

I wonder if we could adapt this admonition for the MySQL case as it appears that the following is True
```
``RemoveField`` will also delete any additional database objects that are
related to the removed field (like views, for example).
```